src/LaravelRequestDocs.php

```diff
@@ -426,6 +460,7 @@ public function rulesByRegex(string $requestClassName, string $methodName): arra
                 return ['key' => $fieldName, 'rules' => $definedFieldRules];
             })
             ->keyBy('key')
+            // @phpstan-ignore-next-line
```

Comment:
This comment was added to skip phpstan, I didn't make changes to this part of the code, I just added the comment so that the pipeline wouldn't break like it was done a few lines above